<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>352</width>
    <height>247</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>352</width>
    <height>247</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>352</width>
    <height>247</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="ok_bt">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>OCR A Extended</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color:rgb(102, 112, 98);
border-radius : 10px;
color:white}</string>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cel_bt">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>200</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>OCR A Extended</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color:rgb(102, 112, 98);
border-radius : 10px;
color:white}</string>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="bg_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>26</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background :qlineargradient(spread:pad, x1:0, y1:0, x2:0.784091, y2:0.733, stop:0 rgba(218, 225, 221, 255), stop:1 rgba(234, 219, 221, 255));border-radius : 20;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_login">
   <property name="geometry">
    <rect>
     <x>-30</x>
     <y>0</y>
     <width>241</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;LOGIN&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-family:'OCR A Extended'; font-size:18pt; font-weight:600;&quot;&gt;Remove File&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="bg_2">
   <property name="geometry">
    <rect>
     <x>-100</x>
     <y>-30</y>
     <width>1531</width>
     <height>841</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>861</width>
     <height>721</height>
    </size>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background :rgb(255, 255, 255);border-radius : 20;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="list_tweet" name="list_remove">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>331</width>
     <height>141</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>OCR A Extended</family>
     <pointsize>11</pointsize>
    </font>
   </property>
  </widget>
  <zorder>bg_2</zorder>
  <zorder>ok_bt</zorder>
  <zorder>cel_bt</zorder>
  <zorder>bg_6</zorder>
  <zorder>label_login</zorder>
  <zorder>list_remove</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>list_tweet</class>
   <extends>QListWidget</extends>
   <header>main_tw.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
